ymcatwincities/openy
# Adjust spacings in the header according to Open Y & YUSA branding

*Steps to review:*
- update the code;
- check that the spacing between the logo and AOF is correct;
- check that top menu and "Open YMCA" text are aligned;
- check that main menu doesn't jump on hover or focus.
